tests/unit/landingai/test_predict.py

```diff
@@ -145,25 +144,6 @@ def test_predict_300():
     )
 
 
-@responses.activate
```

Comment:
we should add an alternative test instead.

Reply:
yes, there is [a way](https://stackoverflow.com/questions/24897145/python-mock-multiple-return-values?noredirect=1&lq=1) to specify different behaviors for a series of calls. Though it's more work to fit into the current test.
maybe we can focus on solving [this one](https://github.com/landing-ai/landingai-python/pull/106#discussion_r1313404230) so we don't have to rework this test.

Reply:
Done.. reverted back to original test